libp2p/7-properties.md

```diff
@@ -190,3 +190,19 @@ For now, we use [protobuf](https://github.com/google/protobuf) for all protocol
 ### 7.5.6 Secure Communications
 
 The wire protocol is -- of course -- wrapped with encryption. We use cyphersuites similar to TLS. This is explained further in the [network spec](./#encryption).
+
+### 7.5.7 Protocol Multicodecs
+
+Here, we present a table with the multicodecs defined for each IPFS protocol that has a wire componenent. This list may change overtime and currently exists as a guide for implementation
+
+| protocol | multicodec |
+| :------: | :--------: |
+| secio    | /ipfs/secio/1.0.0 |
+| plain    | /plaintext/1.0.0 |
+| identify | /ipfs/identify/1.0.0 |
```

Comment:
This has changed to `/ipfs/id/1.0.0`, which is also what I remember from the discussion.  Could you confirm? 

```
p2p/protocol/identify/id.go
25:const ID = "/ipfs/id/1.0.0"
```